indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -78,14 +78,26 @@
                                     {% trans %}n/a{% endtrans %}
                                 {% endif %}
                             </td>
-                            <td class="i-table">
-                                <select>
-                                    <option>--No session--</option>
-                                    {% for section in sections %}
-                                        <option>{{ section.name }}</option>
-                                    {% endfor %}
-                                    <option>Add to new session...</option>
-                                </select>
+                            <td class="i-table" style="white-space: normal;">
+                                {%- set text_color = '#' + contrib.session.colors.text -%}
```

Comment:
i think it is a good idea since we are doing exactly the same in the session list